src/masternodes/mn_checks.cpp

```diff
@@ -2834,9 +2836,9 @@ class CCustomTxApplyVisitor : public CCustomTxVisitor
                 tokenId = tokenDUSD->first;
 
                 // Calculate the DFI amount in DUSD
-                paybackAmount = MultiplyAmounts(dfiUSDPrice, kv.second);
-                if (dfiUSDPrice > COIN && paybackAmount < kv.second)
-                    return Res::Err("Value/price too high (%s/%s)", GetDecimaleString(kv.second), GetDecimaleString(dfiUSDPrice));
+                paybackAmount = MultiplyAmounts(dfiUSDPrice, paybackAmount);
```

Comment:
wouldn't it be clearer to read to make this `paybackAmount = MultiplyAmounts(dfiUSDPrice, tokenLoanBalance);`
Specially with the comparison in the next line?

Reply:
Please ignore. Superceded by 1089